### ISSUE TITLE
testing routes/handlers

### ISSUE THREAD

**Issue by beetleman** [macchiato-framework/macchiato-core#24]
How can I test my routes/handlers? In my opinion something like [this](https://github.com/macchiato-framework/macchiato-core/blob/e46036df3413f9c5aa5024389b95122481ce93b1/test/macchiato/test/mock/request.cljs) should be part of separate lib: `macchiato-testing` or something like this.

**Comment by yogthos** [comment]
Yeah that sounds like a good idea, probably would be good to base it on [ring-mock](https://github.com/ring-clojure/ring-mock/blob/master/src/ring/mock/request.clj).

**Comment by beetleman** [comment]
I need something like ring-mock for my side project so i will develop some kind of mock any way. I'll try do it in way similar to ring-mock. In next weekend I should have some time for this.

**Comment by yogthos** [comment]
Fantastic, that would be a great addition for macchiato. :)
